governance/CHARTER.md

```diff
@@ -21,19 +21,13 @@ b.  The scope of the Project includes collaborative development under the Projec
 
 a.  The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open source Project.
 
-b.  The TSC voting members are initially the Project's Committers. At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project's code repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the CONTRIBUTING file. Any meetings of the Technical Steering Committee will be open to the public, and can be conducted electronically, via teleconference, or in person.
+b.  The TSC voting members are as set forth within the [`README.md` file](README.md) within this repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the [`README.md` file](README.md). Any meetings of the Technical Steering Committee will be open to the public, and can be conducted electronically, via teleconference, or in person.
```

Comment:
I'm removing the "are initially the Project's Committers" as I guess that was from when the TSC was initially setup and it's no longer needed.